torch/onnx/symbolic_opset10.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Which CI checks fail if this is not added?

Reply:
flake8-py3 error in https://github.com/pytorch/pytorch/runs/4476343892?check_suite_focus=true
./torch/onnx/symbolic_opset10.py:1:1: C101 Coding magic comment not found
Please fix the above Flake8 warnings.
Error: Process completed with exit code 1.